.github/workflows/get-gke-credentials-it.yml

```diff
@@ -0,0 +1,121 @@
+name: get-gke-credentials Integration
+
+on:
+  push:
+    paths:
+    - 'get-gke-credentials/**'
+    - '.github/workflows/get-gke-credentials**'
+  pull_request:
+    paths:
+    - 'get-gke-credentials/**'
+    - '.github/workflows/get-gke-credentials**'
+
+jobs:
+  gcloud:
+    name: with setup-gcloud & kubectl
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master # Set up ADC for credentials
+      with:
+        service_account_key: ${{ secrets.GET_GKE_CRED_SA_KEY_JSON }}
+        export_default_credentials: true
+    - id: build
+      name: Build dist
+      working-directory: get-gke-credentials
+      run: |-
+        npm install
+        npm run build
+    - id: get-kubeconfig
+      uses: ./get-gke-credentials
+      with:
+        project_id: ${{ secrets.GET_GKE_CRED_PROJECT }}
+        name: ${{ secrets.GET_GKE_CRED_CLUSTER_NAME }}
+        location: ${{ secrets.GET_GKE_CRED_CLUSTER_LOCATION }}
+        use_auth_provider: true
+    - name: test-kubectl
+      run: |
+        pods=$(kubectl get pods -o=jsonpath='{$.items[*]}')
```

Comment:
nit: add a comment for what this is testing

Reply:
Done